### PR TITLE
MERL-641: Update faustwp to faust for secret key in env file

### PR DESCRIPTION
**Pull request by matthewguywright**
Update faustwp to faust for secret key in env file

### CODE REVIEW

**Review by theodesp — DISMISSED** [review]

**Comment by TeresaGobble** [comment]
Thank you for making this change for consistency's sake!

**Review by TeresaGobble — DISMISSED** [review]
Just an extra space on line 5 in .env.local.sample that is failing the linting test I believe. After that's fixed, this looks good to me! Thanks Matt!

**Comment by matthewguywright** [comment]
NOTE: THIS IS NOT TO BE MERGED UNTIL AFTER RELEASE TUESDAY

**Review by TeresaGobble — APPROVED** [review]
There is a linting error here, but I think it's an issue with the github change tracking. I'm looking at your code here and there is no extra line: 
image

I agree with @blakewilson that this should just be merged as is. Thank you Matt!

File: .env.local.sample [hunk 1]
```diff
@@ -2,4 +2,4 @@
 NEXT_PUBLIC_WORDPRESS_URL=https://atlasportfolio.wpengine.com
 
 # Plugin secret found in WordPress Settings->Headless
-# FAUSTWP_SECRET_KEY=YOUR_PLUGIN_SECRET
+FAUST_SECRET_KEY=YOUR_PLUGIN_SECRET
```
Comment (josephfusco) [inline .env.local.sample:5]:
```suggestion
FAUST_SECRET_KEY=YOUR_PLUGIN_SECRET
```

**Review by blakewilson — APPROVED** [review]
@matthewguywright Looks great Matt!

**Review by josephfusco — APPROVED** [review]
🔥

**Pull request merged**